tests/test_searcher.py

```diff
@@ -0,0 +1,79 @@
+from mock import patch
+import unittest
+
+from search.searcher import Searcher
+
+
+class SearcherTests(unittest.TestCase):
+
+    # Argument values
+    REPO_NAME = 'haiti'
+    MAX_RESULTS = 10
+    # Return values
+    EXTERNAL_SEARCH_RETURN_VALUE = ['external search return value']
+    FULLTEXT_RETURN_VALUE = ['full-text return value']
+    INDEXING_RETURN_VALUE = ['indexing return value']
+
+    def setUp(self):
+        pass
+
+    @patch('external_search.search')
+    def test_external_search_backend_results(self, external_search_mock):
+        """Return external search backend results when available."""
+        external_search_mock.return_value = (
+            SearcherTests.EXTERNAL_SEARCH_RETURN_VALUE)
+        external_search_value = 'abc external search'
+        searcher = Searcher(
+            SearcherTests.REPO_NAME, external_search_value, False,
+            SearcherTests.MAX_RESULTS)
+        assert (searcher.search({'name': 'matt'}) ==
+                SearcherTests.EXTERNAL_SEARCH_RETURN_VALUE)
+        assert 1 == len(external_search_mock.call_args_list)
```

Comment:
Swap the left hand side and the right hand side for consistency (actual == expected).